pkgs/development/python-modules/recipe-scrapers/default.nix

```diff
@@ -64,7 +64,7 @@ buildPythonPackage rec {
   meta = with lib; {
     description = "Python package for scraping recipes data";
     homepage = "https://github.com/hhursev/recipe-scrapers";
-    changelog = "https://github.com/hhursev/recipe-scrapers/releases/tag/${version}";
+    changelog = "https://github.com/hhursev/recipe-scrapers/releases/tag/${src.tag}";
```

Comment:
Unless I'm mistaken, #367739 is still open. I'm opposed to using `src.tag` (even if `buildPythonPackage` doesn't _yet_ support `finalAttrs`).